geometry_msgs/PoseStamped object_pose
---
string result
bool grasp